include/hse/hse.h

```diff
@@ -560,6 +560,18 @@ hse_kvs_get(
     size_t               buf_len,
     size_t *             val_len);
 
+/** @brief Get the name of a KVS.
+ *
+ * @param kvs: KVS handle from hse_kvdb_kvs_open()
+ *
+ * @note This function is thread safe.
+ *
+ * @returns KVS name.
+ * @retval NULL if given an invalid KVS handle.
+ */
+HSE_EXPORT const char *
+hse_kvs_name_get(struct hse_kvs *kvs);
+
```

Comment:
I really dislike the attributes at the beginning of function definitions...

Reply:
When I did this, I just followed patterns I had seen in other projects. Feel free to submit a PR to "fix" this.